### DISCUSSION TITLE
Firebase-ui & alpha-sdk

### DISCUSSION THREAD

**Discussion by HugoGresse** [firebase/firebase-js-sdk#4375] [Q&A] [answered]
Hey there,

What would you recommand to take advantages of the new alpha SDK if our app is using firebase-ui (or the react alternative)?

**Comment by davideast** [comment, marked as answer]
Hey @HugoGresse! We don't have a version of Firebase UI available for the new SDK just yet. But hang tight!

**Reply by benaltair** [reply]
The docs currently read:
> The v9 compat library permits its usage alongside v9, but without the app size reduction and other benefits of the v9 SDK.

https://firebase.google.com/docs/web/modular-upgrade#benefits

Do you have any thoughts @davideast on how to use `import "firebase/compat/*` with the current FirebaseUI library? It's not clear to me so far.

**Reply by Feiyang1** [reply]
@sam-gc

**Reply by sam-gc** [reply]
Unfortunately Firebase UI imported from NPM doesn't work with the compatibility package yet. It does work with Firebase UI pulled in from the CDN, however: https://github.com/firebase/firebaseui-web#option-1-cdn

(If you're using the firebase compat SDK from npm as well, you'll need to put the `firebase` variable onto the window object so that Firebase UI knows where to find it)

**Reply by benaltair** [reply]
Thanks @sam-gc

**Reply by jamesdaniels** [reply]
@benaltair if you're feeling up trying it out, I've started work on making firebaseui compatible with non-CDN `firebase/compat/*` over at https://github.com/firebase/firebaseui-web/pull/850

**Reply by benaltair** [reply]
Great @jamesdaniels I'll consider trying this out over the next couple weeks.

**Reply by jamesdaniels** [reply]
Release candidate now available `npm i --save firebaseui@next firebase@next`
